### DISCUSSION TITLE
clean.conc.blq: feats for tmax reference and default keep

### DISCUSSION THREAD

**Discussion by Gero1999** [humanpred/pknca#340] [Ideas]
Hello, thank you for this great package! I would like to propose two potential features for this function:

- **Add before.tmax & after.tmax in `conc.blq` list argument.** In NCA is very typical to use `tmax` as a reference on how to deal with BLQ data. I think that it would be very easy to change PKNCA `clean.conc.blq` to include for the list definition "before.tmax"and "after.tmax" to make this possible, but please let me know what you think! 

- **Default keep values in `conc.blq` list argument .** I feel that when a user does not define a specific scenario in the list argument (i.e, middle) they simply don't want to perform any action. For that reason, I believe it would make sense to change `PKNCA.options` to put keep as default when an element list is missing.

Let me know what you think of these requests, I will also try to link a PR with a potential solution. Feel free to give it a look, change or ignore it! 

Pull request in #341

**Comment by billdenney** [comment]
Thanks for the suggestion. It makes sense to me. I've started an issue (#343) for discussion of the implementation.
